### PR TITLE
Adding Docker version to mail for rhel7

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Adding Docker version for RHEL 7 and Ceph 4.3 builds 

Logs for 5.0 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7QVMRU
[Version_info.json file](http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7QVMRU/version_info.json)

Logs for 4.3 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F5IZZ1
[Version_info.json](http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F5IZZ1/version_info.json)

Signed-off-by: Amarnath K <amk@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -932,12 +932,23 @@ def collect_recipe(ceph_cluster):
     """
     version_datails = {}
     client_node = ceph_cluster.get_ceph_objects("client")
```
Comment (psathyan) [inline run.py:934]: Please switch to the installer and pull the required details

File: run.py [hunk 2]
```diff
@@ -932,12 +932,23 @@ def collect_recipe(ceph_cluster):
...
         sudo=True, cmd="podman --version | awk {'print $3'}"
     )
     output = out.read().decode().rstrip()
-    log.info(f"Podman Version {output}")
-    version_datails["PODMAN"] = output
+    if output:
+        log.info(f"Podman Version {output}")
+        version_datails["PODMAN"] = output
+
+    out, rc = client_node[0].exec_command(
```
Comment (psathyan) [inline run.py:944]: Add `check_ec=False` as it would raise an exception... the environment could have either of them.

File: run.py [hunk 3]
```diff
@@ -932,12 +932,23 @@ def collect_recipe(ceph_cluster):
...
+        sudo=True, cmd="docker --version | awk {'print $3'}"
+    )
+    output = out.read().decode().rstrip()
+    if output:
+        log.info(f"Docker Version {output}")
+        version_datails["DOCKER"] = output
+
     out, rc = client_node[0].exec_command(
         sudo=True, cmd="ceph --version | awk {'print $3'}"
```
Comment (psathyan) [inline run.py:953]: In case of 5.x, you would need to execute this command in `cephadm` if we plan to move to using the installer node.

Comment (AmarnatReddy) [reply run.py]: Hi Satya,
In both 4.x and 5.0 we are not installing ceph-common on the installer node.
So for getting ceph version I am using the client node.
for the rest of the things I have used the Installer node

**Review by psathyan — APPROVED** [review]

**Pull request merged**